hack/e2e-common.sh

```diff
@@ -334,5 +335,5 @@ function get_vips() {
   fi
 
   vip=$(sed -n "${idx}p" "$vips")
-  echo "$vip"
+  printf "%s" "$vip"
```

Comment:
nit: the format string is redundant
```suggestion
  printf "$vip"
```

...but for that matter, so is the temporary variable. We could just consolidate with the previous line:
```suggestion
  sed -n "${idx}p" "$vips"
```